lib/mixins/lambdas.js

```diff
@@ -5,6 +5,12 @@ module.exports = function lambdas(req, res) {
     renderField() {
       return function renderFieldMixin() {
         const mixin = this.mixin;
+        if (this.useWhen) {
+          if (req.sessionModel.get(this.useWhen.field) !== this.useWhen.value) {
```

Comment:
just a thought, could the value be implicitly `true` if not specified, or is that not useful in most cases?